README.md

```diff
@@ -2,25 +2,14 @@
 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/527e527a99244e0ba82fca9e5c470131)](https://app.codacy.com/gh/BuildForSDG/Team-196-Project?utm_source=github.com&utm_medium=referral&utm_content=BuildForSDG/Team-196-Project&utm_campaign=Badge_Grade_Settings)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Name of Project is TBD
 
-Things you may want to cover:
+* This is an api to match users and specialists.
+* It is a collaborative project for TEAM-196 in the BUILD FOR SDG 2020 cohort 1.
 
-* Ruby version
+Team Members Include :
 
-* System dependencies
+- Daniel Dennis
+- Simon Wathigo
+- Daniel Showatonwa
```

Comment:
Shotonwa not Showatonwa

Reply:
I corrected your name in a new commit, then removed the names because the Readme was failing codacy tests. 
I will add your names again.

Reply:
Codacy is saying you should add two spaces before the names
i.e 
### Header
  - me

Reply:
Let me change that real quick.